### ISSUE TITLE
32bit version?

### ISSUE THREAD

**Issue by achingbrain** [scratux/scratux#9]
Is there a recent version of Scratux that will run on Edubuntu on a 32 bit Atom Netbook?

My friend’s son is getting into Scratch but is finding it hard to follow the tutorials with 1.4x which is all that seems to be available in the default apt registry but trying to install the current version or build it from source seems to require a 64 bit system.

**Comment by ShadowXeldron** [comment]
Sorry, 32bit is pretty much dead by now, since most computers nowadays use 64bit. Also, Edubuntu is also dead and is no longer receiving updates and should be considered insecure. I'd suggest trying the Raspberry Pi Foundation's version of Raspbian for PCs: [https://www.raspberrypi.org/downloads/raspberry-pi-desktop/](url)

Raspbian is the only Linux distro to officially support Scratch 3.0 is available for it but it might be very slow on their ancient old netbook.

With that said, the Scratch website works perfectly fine on Linux, as it is in the web browser rather than a desktop app.

**Comment by joancipria** [comment]
Hi @achingbrain , as @Dragon0307 said, the 32 bits platform is too much old to offer support and would require an extra effort that I can't asume. Furthermore, most linux distros are dropping 32 bits so  please consider going for the web version if you need it. Closing issue, feel free to reopen it if you have any questions.

**Comment by achingbrain** [comment]
Fair enough, old hardware is old and people’s time is limited. Thanks anyway.
